lib/cog/adapters/slack/config.ex

```diff
@@ -0,0 +1,7 @@
+defmodule Cog.Adapters.Slack.Config do
+  use Cog.Adapters.Config,
+    key: Cog.Adapters.Slack,
+    schema: [api:
+              [{:token, [:required], :api_token},
+               {:cache_ttl, [:integer], :api_cache_ttl}]]
+end
```

Comment:
`cache_ttl` is not required since we have a default and `token` is a string which we don't have a coercion for. I think the config stuff for the adapter might be able to be rewritten with the json schema things once that's figured out.